datasetinsights/dashboard.py

```diff
@@ -64,14 +67,26 @@ def main_layout(data_root):
 @app.callback(
     Output("main_page_tabs", "children"), [Input("page_tabs", "value")]
 )
-def render_content(tab):
-    if tab == "dataset_overview":
-        return overview(data_root)
-    elif tab == "object_detection":
+def render_content(value):
+    """ Method for rendering dashboard layout based
+        on the selected tab value."""
+    if value == "dataset_overview":
+        return overview.html_overview(data_root)
```

Comment:
It's still confusing how the local variable `data_root` is available here. Can you make sure to read this value by calling `get_data_root` method? Can you add comments in code to explain this a little bit?